strings/ion_laws.json

```diff
@@ -353,6 +359,7 @@
         "STUN BATONS",
         "SUITS",
         "SUNGLASSES",
+	"SUPERMATTER SHARDS",
```

Comment:
no I mean like, the generated ion laws will be like "THERE ARE THREE BAWDY SUPERMATTER ON THE STATION" so it doesn't work in that sense